tests/components/simplefin/test_sensor.py

```diff
@@ -0,0 +1,52 @@
+"""Test SimpleFin Sensor with Snapshot data."""
+
+from datetime import timedelta
+from unittest.mock import AsyncMock, patch
+
+from freezegun.api import FrozenDateTimeFactory
+import pytest
+from syrupy import SnapshotAssertion
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, async_fire_time_changed, snapshot_platform
+
+
+@pytest.mark.usefixtures("entity_registry_enabled_by_default")
+async def test_all_entities(
+    hass: HomeAssistant,
+    snapshot: SnapshotAssertion,
+    mock_config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+    mock_simplefin_client: AsyncMock,
+) -> None:
+    """Test all entities."""
+    with patch("homeassistant.components.simplefin.PLATFORMS", [Platform.SENSOR]):
+        await setup_integration(hass, mock_config_entry)
+
+    await snapshot_platform(hass, entity_registry, snapshot, mock_config_entry.entry_id)
+
+
+async def test_create_entities(
+    hass: HomeAssistant,
+    snapshot: SnapshotAssertion,
+    mock_config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+    mock_simplefin_client: AsyncMock,
+    freezer: FrozenDateTimeFactory,
+) -> None:
+    """Test enties get created."""
+    with patch("homeassistant.components.simplefin.PLATFORMS", [Platform.SENSOR]):
+        await setup_integration(hass, mock_config_entry)
+
+    assert len(hass.states.async_all()) == 8
+
+    freezer.tick(timedelta(minutes=1))
+    async_fire_time_changed(hass)
+    await hass.async_block_till_done()
+
+    assert len(hass.states.async_all()) == 8
```

Comment:
What are we testing here? We already test if the entities are created in the test above, here we test that time passes and the entities are still available?

Reply:
Nothing... REMOVED